lib/hal/font_sasteroids2.h

```diff
@@ -3,6 +3,7 @@
 // www.svgalib.org site. Graciously provided
 // by John Remyn, originally distributed
 // with sasteroids.
+__attribute__((deprecated("deprecated as of 05/2021", "Use font_terminal.h or font_unscii_16.h instead")))
```

Comment:
`font_unscii_16.h` is not a drop-in replacment, because it has different height.

Reply:
Unresolved, because there's no comment saying how this was resolved. We should not resolve discussions unless there's also a comment which concludes the discussion.

- If you changed the code to resolve this, mention what change you did.
- If you think the comment doesn't deserve further discussion (why not?) then state your reasons and let the original commenter decide if they consider it resolved.

Reply:
Resolved, as per @Teufelchen1 comment in https://github.com/XboxDev/nxdk/pull/458#discussion_r618488929:

> Resolved by removing font_sasteroids2.h

*(Please make sure each discussion has a copy of such information when resolving them, otherwise it's very hard to figure out what happened)*